components/brave_wallet_ui/utils/coin-market-utils.ts

```diff
@@ -18,8 +18,14 @@ export const sortCoinMarkets = (
   columnId: MarketGridColumnTypes
```

Comment:
Probably we should update the type of `columnId` to be `(keyof CoinMarket) & MarketGridColumnTypes` but that caused issues at callsites